### PR TITLE
add mosquitto

### PR DESCRIPTION
**Pull request by tprelog**
https://mosquitto.org/

### CODE REVIEW

File: mosquitto.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "mosquitto",
+  "release": "11.3-RELEASE",
+  "artifact": "https://github.com/tprelog/iocage-mosquitto.git",
+  "official": "false",
+  "properties": {
+    "allow_raw_sockets": "1",
+    "boot": "on",
```
Comment (miwi-fbsd) [inline mosquitto.json]: Please let it optional

File: mosquitto.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    "bpf": "on",
```
Comment (miwi-fbsd) [inline mosquitto.json]: Please remove it automated now.

File: mosquitto.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    "dhcp": "1",
+    "vnet": "yes"
```
Comment (miwi-fbsd) [inline mosquitto.json]: Please remove it automated now.

**Comment by miwi-fbsd** [comment]
Hi,

Here another fix
https://github.com/tprelog/iocage-mosquitto/pull/2
https://github.com/tprelog/iocage-mosquitto/pull/3

**Comment by tprelog** [comment]
done

**Pull request merged**